src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -421,7 +421,7 @@
       <UpToDateCheckBuilt Condition="'$(_DebugSymbolsProduced)'=='true' and '$(SkipCopyingSymbolsToOutputDirectory)' != 'true' and '$(CopyOutputSymbolsToOutputDirectory)' != 'false'" Include="@(_DebugSymbolsOutputPath)"/>
 
       <!-- app.config -->
-      <UpToDateCheckBuilt Condition=" '@(AppConfigWithTargetPath)' != '' " Include="@(AppConfigWithTargetPath->'$(OutDir)%(TargetPath)')" Original="@(AppConfigWithTargetPath)"/>
+      <UpToDateCheckBuilt Condition=" '@(AppConfigWithTargetPath)' != '' " Include="@(AppConfigWithTargetPath->'$(OutDir)%(TargetPath)')" Original="%(AppConfigWithTargetPath.OriginalItemSpec)"/>
```

Comment:
Is `OriginalItemSpec` always set on these items?

Reply:
if the configuration files is different than "App.config", the design-time build only will show the correct file name in PrimaryList, and CandidateAssemblyFiles. How to get access to configuration name ? It seems that those variables are not visible for  CollectUpToDateCheckBuiltDesingTime?
The image below is using App66.config 
![image](https://user-images.githubusercontent.com/510598/76923878-e7662880-6891-11ea-9365-fc6540f32b47.png)

Reply:
If we add App1.config, App2.config and App.config only the App.config is taking into consideration to generate the Output file. It seems that we should only care about App.config filename

Reply:
@callesp I'm still experiencing issues with FUTD checking and would like to log a new issue. What tool did you use to generate the [image](https://user-images.githubusercontent.com/510598/76923878-e7662880-6891-11ea-9365-fc6540f32b47.png)? I'd like to do the same when I submit the issue. Thanks.

Reply:
@igitur that screenshot is from the msbuild structured log viewer, showing a binlog captured via the Project System Tools extension.